myhpi/settings.py

```diff
@@ -251,3 +251,13 @@
 
     INSTALLED_APPS += ["myhpi.tenca_django"]
     MIDDLEWARE += ["myhpi.tenca_django.middleware.TencaNoConnectionMiddleware"]
+
+INSTITUTION_EMAIL_REPLACEMENTS = [
```

Comment:
i think we are already dealing with replacements when processing the OIDC login. could you have a look whether there is any chance to reduce code duplication?

Reply:
I think our data should be clean (without uni-potsdam), but the mailing lists may have entries with uni-potsdam, so we need this to access them.